File: A0302/Atividade A0302a/conta.c
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
#include "conta.h"

float abrirConta(dadoConta *c) {
    c->saldo = 0;
    printf("Olá %s o número de sua conta é %d e seu saldo atualmente é %2.0f", c->nomeCliente, c->numConta, c->saldo);

}

float lancaCredito(dadoConta *c) {
    float soma;
    printf("Informe um valor para creditar: ");
    scanf("%f", &soma);
    c->saldo = c->saldo + soma;
    return printf("Seu é saldo atual é %2.0f", c->saldo);
}

float lancaDebito(dadoConta *c) {
    float debito;
    printf("Informe um valor para debitar: ");
    scanf("%f", &debito);
    c->saldo = c->saldo - debito;
    return printf("Seu é saldo atual é %2.0f", c->saldo);
}

void saldo(dadoConta* c) {
    printf("\nCliente: %s", c->nomeCliente);
    printf("Conta..: %d", c->numConta);
    printf("\nSaldo..: %2.0f", c->saldo);

}

File: ContaBancaria25.09/src/contabancaria25/pkg09/Conta.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package contabancaria25.pkg09;

import javax.swing.JOptionPane;

/**
 *
 * @author Alexandre
 */
public class Conta {

    private String nome;
    private int agencia;
    private int numero;
    protected float saldo;

    public Conta() {
        nome = new String();
        agencia = 0;
        numero = 0;
        saldo = 0;
    }

    public Conta(String nome, int agencia, int numero, float saldo) {
        this.agencia = agencia;
        this.nome = nome;
        this.numero = numero;
        this.saldo = saldo;
    }

    public void Depositar(float valor) {
        if (valor < 0) {
            JOptionPane.showMessageDialog(null, "Insira um número positivo");
        } else {
            saldo += valor;
        }
    }

    public void Sacar(float valor) {
        if (valor > saldo) {
            JOptionPane.showMessageDialog(null, "Erro");
        } else {
            saldo -= valor;
        }
    }

    public void MostrarSaldo() {
        JOptionPane.showMessageDialog(null, "Seu saldo podre é:" + saldo);
    }

}
